### ISSUE TITLE
With Rustic, Icons and Type information disappear after changing selected completion.

### ISSUE THREAD

**Issue by asdfer-1234** [jcs-elpa/company-fuzzy#40]
# Steps to reproduce

1. Use the following minimal config as `init.el`

```elisp
(require 'package)
(setq package-archives
      '(("gnu" . "http://elpa.gnu.org/packages/")
        ("melpa" . "https://melpa.org/packages/")))
(package-initialize)

(package-install 'rustic)
(package-install 'company)
(package-install 'company-fuzzy)
(require 'rustic)
(require 'company)
(require 'company-fuzzy)

(with-eval-after-load 'company-fuzzy
  (add-hook 'company-mode-hook 'company-fuzzy-mode))
```

2. Open a rust file within a cargo project.
3. Install `lsp-mode` when prompted.
4. Initiate autocompletion with at least 2 completions available
5. Press down arrow, changing the selection

# Expected behaviour
Next completion gets selected, and icons and type informations remain.

# Actual behaviour
Next completion gets selected, icons and type informations disappear. Revealing a new completion by scrolling the completion menu far enough will result in the just-revealed-by-scrolling completion entry having the icon, and scrolling another entry down will remove the icons from it. Below video demonstrates it.

https://github.com/jcs-elpa/company-fuzzy/assets/58357836/9f8a4b75-369d-4b65-83f0-f519ed1fb26b

**Comment by jcs090218** [comment]
I don't think this error is related to this package since `company-fuzzy` doesn't impact any frontend code. 🤔 

I tested it with `emacs -q`, and I cannot reproduce this bug.

![Screenshot 2024-04-06 143541](https://github.com/jcs-elpa/company-fuzzy/assets/8685505/5d627439-9e48-46a5-8275-aed9dd032d0f)
